DashKit/Mocks/MockPodStatus.swift

```diff
@@ -321,7 +330,13 @@ extension MockPodStatus: RawRepresentable {
 
         if let alarmReferenceCode = rawValue["alarmReferenceCode"] as? String {
             self.alarmReferenceCode = alarmReferenceCode
-        }        
+        }
+        
+        if let lowReservoirAlertRaw = rawValue["lowReservoirAlert"] as? LowReservoirAlert.RawValue, let lowReservoirAlert = LowReservoirAlert(rawValue: lowReservoirAlertRaw)
+        {
```

Comment:
[nit] `{` on the same line as `if`.

Reply:
If the line goes long and wraps, it becomes harder to tell where the start of the block is vs additional conditions, so I tend to put starting brace on next line if long, or multiline.